2wr-app/src/router/index.js

```diff
@@ -32,6 +34,17 @@ const routes = [
       component: EmergencyKitCreatePage,
       meta: { requiresAuth: true }
     },
+    {
+      path: '/prepare/hazardhunt',
+      name: 'hazardhunt',
+      component: HazardHuntListing,
+      meta: { requiresAuth: false }
```

Comment:
```suggestion
      meta: { requiresAuth: true }
```

Looks like the API requires authentication for getting hazard hunt listings so we should set `requiresAuth` to true. That way users will be redirected to login before they can navigate to this route.